docs/quickstart/csharp.md

```diff
@@ -85,16 +89,16 @@ From the `examples/csharp/helloworld-from-cli` directory:
 > dotnet build **/project.json
 ```
 
-### Using Xamarin Studio or Monodevelop IDEs
-The C# gRPC package currently depends on System.Interactive.Async 3.0.0, which cannot be downloaded on older NuGet versions.
-NuGet is too old in Xamarin Studio on OS X and Monodevelop on Linux as well as via apt-get.
+### Using the Monodevelop IDE
+The C# gRPC package currently depends on System.Interactive.Async 3.0.0, which requires NuGet 2.12+ to install.
+The NuGet included on the latest versions of Monodevelop is too old to install gRPC C#.
```

Comment:
what does it mean? that with mono and nuget 2.12 I cannot do the initial restore directly from the monodevelop IDE, but after running 'nuget restore' from the commandline, everything will work as expected when I work with monodevelop afterwards? (if so, things are totally fine I'd say).
We should clarify this.

I'll be fine with running 'nuget restore' from the commandline, but I cannot hit the "restore" button inde tfrom IDE?